apps/web/app/api/comments/validation.ts

```
@@ -0,0 +1,106 @@
+import { z } from 'zod'
+import { createSupabaseServerClient } from '@packages/lib/supabase-server'
```

Biome (2.1.2)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)